app/forms/refunds/original_case_details_form.rb

```diff
@@ -57,7 +57,7 @@ class OriginalCaseDetailsForm < Form
 
     def transfer_name
       claimant_name = [
-        resource.applicant_title.titleize,
+        resource.applicant_title&.titleize,
```

Comment:
What is the & for?

Reply:
It allows for applicant_title being nil so titleize doesnt get called on nil

Reply:
ok so it's a shortcut for try.